manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: Lego
+PackageName: Mindstorms Edu NXT Software

```

This must match control panel
```suggestion
PackageName: LEGO MINDSTORMS Edu NXT Software
```